hoomd/ParticleData.h

```diff
@@ -960,6 +955,12 @@ class PYBIND11_EXPORT ParticleData
         */
         void setFlags(const PDataFlags& flags) { m_flags = flags; }
 
+        /// Enable pressure computations
+        void setPressureFlag()
```

Comment:
Why does this signature differ from the one in `System.h`? It doesn't look like the flag can be disabled here.

Reply:
The `System` object resets the flags in `ParticleData` each timestep and `System::setPressureFlag` controls whether the pressure computation flag is always set or not. This will not take effect until at least one timestep has been run.

`ParticleData::setPressureFlag` takes effect immediately and exists so that users can request the pressure flag and then request the per particle virials from a force compute object without calling `System::run`. The flags are an indication of what data is valid, once the virials have been computed the user cannot remove them so there is no way to immediately remove the flag.